content/docs/v0.4.0/health-rules.md

```diff
@@ -0,0 +1,148 @@
+# Health Rules
+
+Cartographer reports the health of each object stamped on the cluster. If no health rule is specified in the template
+that created the object, Cartographer will report that the object is healthy. By defining one of several types of health
```

Comment:
It's more complicated when no health rule is specified. The object must be stamped successfully and it produces an output, unless it's a ClusterTemplate which is exempt from this last rule.